clients/client-python/tests/integration_test_env.py

```diff
@@ -0,0 +1,88 @@
+"""
+Copyright 2024 Datastrato Pvt Ltd.
+This software is licensed under the Apache License version 2.
+"""
+
+import os
+import unittest
+import subprocess
+import time
+import requests
+
+from gravitino import GravitinoClient
+from gravitino.constants import TIMEOUT
+
+
+def check_gravitino_server_status():
+    try:
+        response = requests.get("http://localhost:8090/api/version")
+        response.raise_for_status()  # raise an exception for bad status codes
+        response.close()
+        return True
+    except requests.exceptions.RequestException as e:
+        print("Failed to access the server: {}".format(e))
+        return False
+
+
+# Provide real test environment for the Gravitino Server
+class IntegrationTestEnv(unittest.TestCase):
+    GravitinoHome = None
+
+    @classmethod
+    def setUpClass(cls):
+        print("Starting integration test environment...")
+        cls.GravitinoHome = os.environ.get('GRAVITINO_HOME')
+        if cls.GravitinoHome is None:
```

Comment:
is that possible to set the default value for `GRAVITINO_HOME` and `PROJECT_VERSION` on the setup?

Reply:
Yes, I set these environment variable in the `build.gradle.kts`, But it only valid in the gradle command, like this `./gradlew :clients:client-python:integrationTest`, If you directly execute integration test in the IDEA, you need manual set in your IDEA or OS.

Reply:
@xunliu If we move the logic that sets the environment from `build.gradle.kts` to `setUpClass`, does it work on my IDEA or OS? does it depend on `compileDistribute`? So we have to set them on the `build.gradle.kts`.

Reply:
> @xunliu If we move the logic that sets the environment from `build.gradle.kts` to `setUpClass`, does it work on my IDEA or OS? does it depend on `compileDistribute`? So we have to set them on the `build.gradle.kts`.

Peidian, I think Xun's point is, you can set these two environment variables in IDE as well; No need to configure them in the test case's codes;

Reply:
> @xunliu If we move the logic that sets the environment from `build.gradle.kts` to `setUpClass`, does it work on my IDEA or OS? does it depend on `compileDistribute`? So we have to set them on the `build.gradle.kts`.

@coolderli you can set these two environment variables in IDE as well;